config/locales/en.yml

```diff
@@ -102,19 +102,30 @@ en:
   projects:
     wait-for-uploads-to-complete: Please wait for your images to finish uploading before submitting your application.
     project:
-      awesome: Awesome
       delete: Delete this project
       see-the-rest: See the rest →
-      view-public-page: View the public page →
-      edit: Edit Project →
-      confirm-delete: Really delete this?
       winner: "Winner for %{name}"
+      hide: Hide
+      unhide: Unhide
+      consider-hiding: Not an appropriate grant?
+      hide-reason: Why is this not appropriate?
+      hide-reason-required: You must supply a reason a to hide a project!
+      hidden-details: "Hidden by %{username} on %{hidden_at} because %{reason}."
+      hiding-explanation: "Hiding a project will hide its details from everyone; the title and why it was hidden will still be visible, and it can be easily unhidden."
+    project_details:
+      images: "Images:"
+      image: "Image #%{index}"
+      awesome: Awesome
       about-me: "A little about me:"
       about-project: "Here's my idea:"
       use-for-money: "How I will use the money:"
       not-answered: Not Answered
-      images: "Images:"
-      image: "Image #%{index}"
+      confirm-delete: Really delete this?
+      view-public-page: View the public page →
+      edit: Edit Project →
+    project_hidden:
+      confirm-delete: Really delete this?
+      view-public-page: View the public page →
```

Comment:
Perfect. I couldn't actually find documentation on how to change the scope (clearly didn't look hard enough), hence the documentation. I'll update that in an hour or so so there's only one copy of each.

On Sun, Nov 1, 2015 at 12:49 PM, Jesse Chan-Norris
<notifications@github.com> wrote:

>>        about-me: "A little about me:"
>>        about-project: "Here's my idea:"
>>        use-for-money: "How I will use the money:"
>>        not-answered: Not Answered
>> -      images: "Images:"
>> -      image: "Image #%{index}"
>> +      confirm-delete: Really delete this?
>> +      view-public-page: View the public page →
>> +      edit: Edit Project →
>> +    project_hidden:
>> +      confirm-delete: Really delete this?
>> +      view-public-page: View the public page →
> This duplication of translation strings seems weird. Why was this broken out into these two sections? If it's because of lazy interpolation, I would instead make the calls as `t(".view-public-page", :scope => "projects.project")` and then use the same translation in both places. Since these are actually the same string being displayed in the same place (not just the same translation of 2 different strings) I think this is more correct.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/awesomefoundation/awesomebits/pull/202/files#r43585821